Lucid/Utils/AnySequence.swift

```diff
@@ -61,6 +61,8 @@ extension AnySequence: Equatable where Element: Equatable {
     }
 }
 
+extension AnySequence: Sendable where Element: Sendable {}
```

Comment:
```suggestion
extension AnySequence: @unchecked Sendable where Element: Sendable {}
```

This line produces warning, please check https://developer.apple.com/documentation/swift/sendable for more details.